make_profiler/parser.py

```
@@ -127,24 +127,45 @@ def replace_include_with_file(
-            })
+        ast.append(
+            (
+                token[0],
+                {
+                    'target': targets[0],
+                    'all_targets': targets,
+                    'grouped': sep == '&:',
+                    'deps': [deps, order_deps],
+                    'docs': docs,
+                    'body': body,
+                }
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)